docs/quickstart.mdx

```diff
@@ -48,43 +34,38 @@ Let's make it happen!
 We're going to use Hardhat, but Sapphire should be compatible with your dev
 environment of choice. Let us know if things are not as expected!
 
-1. Make & enter a new directory
-2. `npx hardhat@~2.19.2 init` then create a TypeScript project.
+1. Make & enter a new directory `mkdir quickstart && cd quickstart`
+2. `npx hardhat@~2.19.2 init` then create a TypeScript project and install the project dependencies.
```

Comment:
Could we sneak in this bump? https://www.npmjs.com/package/@oasisprotocol/sapphire-hardhat?activeTab=versions

```suggestion
2. `npx hardhat@~2.19.4 init` then create a TypeScript project and install the project dependencies.
```